src/screens/Ticketing/Ticket/Details/DetailsScreen.tsx

```diff
@@ -55,6 +61,17 @@ export default function DetailsScreen({navigation, route}: Props) {
             hasActiveTravelCard={hasActiveTravelCard}
           />
         )}
+
+        {firstTravelRight?.type === 'PreActivatedSingleTicket' &&
```

Comment:
Skulle utils i `@atb/tickets` utvides til å ha hjelpefunksjoner for dette også på samme måte som `isPreactivatedTicket`?

Reply:
Det har den allerede ser jeg 🙈